tests/wpt/mozilla/tests/mozilla/cross-origin-objects/cross-origin-postMessage-child1.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
A lot of the tags here (and in the other tests) can go away and would make this a bit less noisy, but your choice :)